### ISSUE TITLE
Invisible geometry importing glTF with more than 4 bone weights

### ISSUE THREAD

**Issue by punto-** [godotengine/godot#56705]
### Godot version

v3.4.3.rc.custom_build [dd002197d] 

### System information

windows 10, gles3

### Issue description

When importing the attached glb, the geometry is invisible, seems to be empty, no vertices, When importing in 3.3 or 4.0, it loads correctly. The model has a skeleton and some animations, both seem to load fine

### Steps to reproduce

import glb into project, load scene

### Minimal reproduction project

[bird.zip](https://github.com/godotengine/godot/files/7850455/bird.zip)

**Comment by fire** [comment]
Can confirm.

![image](https://user-images.githubusercontent.com/32321/149038699-10161234-5b16-4549-b4b1-286c009bd544.png)

The bug happens when the skeleton is at the root.
[bird_rev.zip](https://github.com/godotengine/godot/files/7850509/bird_rev.zip)

**Comment by lyuma** [comment]
Well, I tried the model and got this error:
```
 modules/gltf/gltf_document.cpp:2684 - Condition "a.has("JOINTS_0") && a.has("JOINTS_1")" is true. Continuing.
```

This error means this model is not supported in Godot 3.4, because it uses more than 4 bone weights per vertex.

During the development of the glTF module, I remember that it was decided that models with more than 4 bones could not be reasonably imported (based on some chat conversations I had with @fire and others), because some pathological cases exist which require more than 4 bone weights to function.  My opinion is that the top 4 bones could be selected and renormalized, but I would need to understand more to know if this is a viable approach.

I don't actually know how Godot <=3.3 dealt with models with >4 bone weights, such as JOINTS_0 and JOINTS_1. it's possible it just ignored JOINTS_1, but did it renormalize the remaining bones?

**Comment by fire** [comment]
My last attempt at normalizing bones is here. https://github.com/godotengine/godot/pull/48360

**Comment by fire** [comment]
Discussed what happens when the number of godot weights is above 8 like 16 weights with @punto- 

1. want to use the same algorithm as normalizing 4 weights
2. normalize with the constants of 8 weights to solve more weights 
3. can also make @reduz 's once per frame compute shader support arbitrary weights

**Comment by RockyMadio** [comment]
I can also confirm this. Either some rotation values are added into some bones (usually spines). After reimporting the gltf a few times, all meshes disappear.

**Comment by akien-mga** [comment]
Fixed by #61912.
